packages/flutter_tools/test/integration.shard/test_driver.dart

```diff
@@ -105,7 +105,7 @@ abstract class FlutterTestDriver {
         .followedBy(arguments)
         .toList(),
       workingDirectory: _projectFolder.path,
-      environment: <String, String>{'FLUTTER_TEST': 'true'},
+      environment: <String, String>{'FLUTTER_TEST': 'true', 'FLUTTER_WEB': 'true'},
```

Comment:
Consider documenting here that this achieves the effect of `flutter config --enable-web`.

Reply:
Done